robot/basestation/static/js/themes.js

```diff
@@ -0,0 +1,58 @@
+$(document).ready(() => {
+
+    function append_css(file) {
+        console.log('Append CSS: ' + file);
+
+        var link = document.createElement("link");
+        link.href = '/static/css/' + file + ".css";
+        link.type = "text/css";
+        link.rel = "stylesheet";
+        link.media = "screen,print";
+
+        document.getElementsByTagName("head")[0].appendChild(link);
+    }
+
+    function prefer_theme(name) {
+        console.log('Prefer theme: ' + name);
+        var futuredate = new Date();
+        var futuretime = futuredate.getTime();
+        futuretime += 3600 * 1000 * 24 * 365; // Roughly 1 year into the future
+        futuredate.setTime(futuretime);
+        document.cookie = "theme=" + name + "; expires=" + futuredate.toUTCString() + "; path=/";
+        append_css('themes/' + name);
+    }
+
+    function get_cookie(cname) {
```

Comment:
This function is already defined in `helpers.js`, was there any particular reason for redefining this?

Reply:
A) I didn't technically redefine a function - it has a different name - it has the same functionality - it's technically a duplicate.

B) I hadn't noticed the existence of helpers.js at all. Refactoring could be a TODO item for later.

C) This new get_cookie() is scoped only within the context of $(document).ready(), as far as I can tell, so, it's mostly harmless?

Reply:
Ah I didn't notice the snake_case vs camelCase, fair enough. Nonetheless, since it has the same functionality we might as well use the already existing one. Better to clean this up now than to add duplicate code.